services/user.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User

```

Comment:
You're importing `User` from `django.contrib.auth.models`, but you never use it in your code. It's a good practice to remove unused imports to keep your code clean. Also, according to the checklist, you should use `get_user_model()` instead of `User`.